src/constants/MarkdownFiles/posts/2025-07-14-gsoc-25-BishoyWadea-week06.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "GSoC ’25 Week 06 Update by Bishoy Wadea"
+excerpt: "Mid term evaluation reflection"
+category: "DEVELOPER NEWS"
+date: "2025-07-13"
+slug: "gsoc-25-BishoyWadea-week06"
+author: "@/constants/MarkdownFiles/authors/bishoy-wadea.md"
+tags: "gsoc25,sugarlabs,week06,midterm,BishoyWadea"
+image: "assets/Images/GSOC.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 05 Progress Report by Bishoy Wadea
+
+**Project:** [Euclid’s Game](https://github.com/Bishoywadea/Euclid-s-Game)  
+**Mentors:** [Ibiam Chihurumnaya](https://github.com/chimosky)  
+**Assisting Mentors:** [Walter Bender](https://github.com/walterbender/)  
+**Reporting Period:** 2025-06-29 - 2025-07-05 
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Start implementing Euclid’s Game
+---
+
+## This Week’s Achievements
+
+### *Goal 1: add helpful video tutorial in Soma Cube Game*
+
+1. **add video in help button**  
+   - commit: [modify code to handle help videos](https://github.com/Bishoywadea/Soma-Cube/commit/63a7daaa8009f5f54791cdf9081e765846135f70)
+![illistration tutorial video](https://github.com/Bishoywadea/Soma-Cube/blob/main/help.mp4)
```

Comment:
The video is not showing up.

Perhaps add it to YouTube and point there?

<img width="1920" height="1080" alt="Screenshot from 2025-07-15 15-55-22" src="https://github.com/user-attachments/assets/c8c9ded5-61a3-4e23-b293-ae8b2d40afb1" />


Also, check your spelling for words like illustration"